go-sms_inbound.js

```diff
@@ -1415,6 +1417,38 @@ go.app = function() {
             });
         });
 
+        self.states.add('state_change_baby', function(name) {
+            return go.utils
+                .get_identity_by_address(
+                    {'msisdn': self.im.user.answers.contact_msisdn}, self.im)
+                .then(function(identity) {
+                    if (identity) {
+                        self.im.user.set_answer('mother_id', identity.id);
+                        return go.utils_project
+                            .switch_to_baby(self.im, self.im.user.answers.mother_id)
+                            .then(function() {
+                                return self.states.create('state_new_registration_baby');
+                        });
+                    } else {
+                        return self.states.create('state_end_unrecognised_baby');
+                    }
+                });
+        });
+
+        self.states.add('state_new_registration_baby', function(name) {
+            return new EndState(name, {
+                text: $("{{prefix}}Thank you. You will now receive messages about caring for the baby").context({prefix:""}),
```

Comment:
Also, is this the correct copy that we received? Seems strange to say "the baby" instead of "your baby" to the mother.